docs/Getting_Started.md

```diff
@@ -1,47 +1,14 @@
-# Prerequisites 
+# Overview
 
-### Scenario Editor
+## Download
 
-1. Ensure that [Visual Studio 2019](https://visualstudio.microsoft.com/downloads/) is installed to generate the project files.
+monoDrive products are available for evaluation as well as access to our open source clients are available by contacting NI's monoDrive support, and the monoDrive Simulator is available for evaluation through [monoDrive Website](https://www.monodrive.io/register/). Inquiries about purchasing and/or acquiring an evaluation can be provided by contacting NI's monoDrive support at <b>support@monodrive.io</b>. 
```

Comment:
This link doesn't actually take you the the registration page, just the home page.